godot_fmod.cpp

```diff
@@ -772,6 +783,8 @@ void Fmod::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("sound_set_pitch", "uuid", "pitch"), &Fmod::setSoundPitch);
 	ClassDB::bind_method(D_METHOD("sound_get_pitch", "uuid"), &Fmod::getSoundPitch);
 
+	ClassDB::bind_method(D_METHOD("set_sound_3d_settings", "dopplerScale", "distanceFactor", "rollOffScale"), &Fmod::setSound3DSettings);
```

Comment:
Use the system namespace in front of the method name. Makes things easier for auto-completion in the script editor.
```suggestion
	ClassDB::bind_method(D_METHOD("system_set_sound_3d_settings", "dopplerScale", "distanceFactor", "rollOffScale"), &Fmod::setSound3DSettings);
```